main/src/ca/uwaterloo/flix/language/phase/TreeShaker2.scala

```diff
@@ -25,11 +25,11 @@ import ca.uwaterloo.flix.util.ParOps
 /**
   * The Tree Shaking phase removes all unused function definitions.
   *
-  * A function is considered reachable if it:
+  * A function is considered entryPoints if it:
```

Comment:
Grammar in this file

Reply:
I think we should still write "reachable"

Reply:
oops, too much trust in intellij refactor